Theme support fixes

DartVanya opened the pull request:
- Visual clarity for disabled buttons
- Theme support for Handle statistics dialog
- [Add theme support for Job page dialogs](https://github.com/winsiderss/systeminformer/pull/2185/commits/55316686b2c98f13ad435d248fef83dc8f8051e4)
- [Add theme support for Memory protection dialog](https://github.com/winsiderss/systeminformer/pull/2185/commits/850a54b5f0ac002a386ac7c643e16b507a9c17b7)
- [Theme support for Column set editor](https://github.com/DartVanya/systeminformer/commit/7146e5ae623b44acdb2e77727b56a71b5316b747)
- [TpWorkerFactory tab theme](https://github.com/winsiderss/systeminformer/pull/2185/commits/9cd91e387c6185e65830e54af4a5696d40ac3ebd)

https://github.com/user-attachments/assets/c796f199-c6d3-4f59-b549-19783a98294a

![image](https://github.com/user-attachments/assets/c8e68d08-04e4-4814-9a92-9bc6f3cc3d52)     ![image](https://github.com/user-attachments/assets/6614654a-b856-47bc-aa7e-889dc0531101)

dmex commented:
> Theme support for Handle statistics dialog (for scrollbar need IAT hook of OpenNcThemeData, because there is no Scroll bar control in this dialog).

Enabling the theme for the listview also enables the scrollbar theme?

DartVanya commented:
> > Theme support for Handle statistics dialog (for scrollbar need IAT hook of OpenNcThemeData, because there is no Scroll bar control in this dialog).
> 
> Enabling the theme for the listview also enables the scrollbar theme?

I moved down theme init line (after whole dialog init code), and the scrollbar also started working. I added theme support for Job dialogs. Video preview in PR description.
